apps/mobile/src/services/tx/tx-sender/sign.ts

```diff
@@ -37,56 +28,29 @@ export const signTx = async ({
   }
 
   const { protocolKit, wallet } = await createConnectedWallet(privateKey, activeSafe, chain)
-  const { safeTx, signatures } = await createExistingTx({
+  const { safeTx } = await createExistingTx({
```

Comment:
Nit: I can't add a comment to the function itself but I'd suggest renaming this to something like `proposeTx` for clarity.